targets/targets.json

```diff
@@ -256,7 +256,7 @@
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-M3",
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM", "GCC_CR", "IAR"],
-        "extra_labels": ["NXP", "LPC176X"],
+        "extra_labels": ["NXP", "LPC176X", "UBLOX_MODEM"],
```

Comment:
Do you really need another `TARGET_` directory here? Could you just put it in the `TARGET_UBLOX_C027` directory?